float64 distance    # Distance. Can be negative or positive.
float64 turn_angle  # Angle to turn
float64 bot_speed   # Maximum _requested_ forward speed. Should always be positive
float64 turn_speed  # Maximum _requested_ turn speed. Should always be positive
float64 wheel_speed # Maximum _requested_ speed for any individual wheel

bool feedback       # Whether or not to use feedback to control the driver

---

float64 distance    # Final distance driven
float64 turn_angle  # Final turn angle

---

float64 distance    # Current distance driven
float64 turn_angle  # Current turn angle
